paratex/__main__.py

```diff
@@ -4,19 +4,48 @@
 import pandas as pd
 from extractor import extract_attendance, Session, fetch_sessions_from_interval
 
-OUTPUT_PATH = Path(__file__).parent / 'output'
+PARTITIONED_PATH = Path(__file__).parent / 'output_partitioned'
+PARTITIONED_PATH.mkdir(parents=True, exist_ok=True)
+JOINED_PATH = Path(__file__).resolve().parent.joinpath('joined_csvs')
+JOINED_PATH.mkdir(parents=True, exist_ok=True)
 
 
-def main():
-    sessions = fetch_sessions_from_interval(date(2011, 10, 15), date(2019, 10, 15))
-
+def partitioned_process(sessions):
```

Comment:
Se for o caso de criar a pasta, faça aqui:

```suggestion
def partitioned_process(sessions):
    PARTITIONED_PATH.mkdir(parents=True, exist_ok=True)
```